build.sh

```diff
@@ -1,123 +1,16 @@
 #!/usr/bin/env bash
 
-# adding dotnet to the path. it is needed to run toolset csc.
-parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-dotnet_path=$parent_path/dotnetcli
-export PATH=$PATH:$dotnet_path
+source="${BASH_SOURCE[0]}"
 
-usage()
-{
-  echo "build.sh [ Debug(default) | Release ]"
-}
+# resolve $SOURCE until the file is no longer a symlink
+while [[ -h $source ]]; do
+  scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+  source="$(readlink "$source")"
 
-if [ "$1" == "-?" ] || [ "$1" == "-h" ]; then
-  usage
-  exit
-fi
+  # if $source was a relative symlink, we need to resolve it relative to the path where the
+  # symlink file was located
+  [[ $source != /* ]] && source="$scriptroot/$source"
+done
 
-if [ $# -eq 0 ]
-  then
-    Configuration="Debug"
-  else
-    Configuration=$1
-fi
-
-BuildVersion="preview2-$(date '+%y%m%d')-1"
```

Comment:
What are the package versioning rules now? Can we control it?

Reply:
We could if we wanted to, but the way this is will now use the same arcade versioning that the rest of the repos use. That is (version)-(prereleaselabel)-(prereleaseRevision)-(arcadegeneratedDateBasedVersion).(BuildRevision)

Reply:
We could if we wanted to, but the way this is will now use the same arcade versioning that the rest of the repos use. That is (version)-(prereleaselabel)-(prereleaseRevision)-(arcadegeneratedDateBasedVersion).(BuildRevision)

Reply:
Will the versions always remain in pre-release/preview? I am asking to make sure we avoid marking them as stable.

Reply:
Yes. You need to specify a property to mark as stable and we can also do that on a per project basis.